Classes/Private/Manager/Objects/OCTActiveFile+Variants.h

```diff
@@ -0,0 +1,58 @@
+//
+//  OCTActiveFile+Variants.h
+//  objcTox
+//
+//  Created by stal on 9/7/2015.
+//  Copyright © 2015 Zodiac Labs. All rights reserved.
+//
+
+#import "OCTSubmanagerFiles.h"
+#import "OCTActiveFile.h"
+
+#define AVERAGE_SECONDS 10
+
+@class OCTMessageFile;
+
+@interface OCTActiveFile ()
+
+@property (weak)      OCTSubmanagerFiles *fileManager;
```

Comment:
To be more consistent with the rest of the project, can you please add property attributes to all properties all over you change (except for Realm objects - Realm ignores those attributes)? Format

```objc
(strong/weak/assign, atomic/nonatomic, [readwrite, readonly])
```